cmd/checkcommits/checkcommits.go

```diff
@@ -146,14 +146,6 @@ func checkCommitBodyLine(config *CommitConfig, commit string, line string,
 		}
 	}
 
-	// Check first character of line. If it's _not_
```

Comment:
just a nit - you could have mentioned 'fixed by swapping the order of the tests' in the commit message. It is sort of implied, but not stated.